src/layouts/EditHomepage/EditHomepage.jsx

```diff
@@ -1551,8 +1566,7 @@ const EditHomepage = ({ match }) => {
                                       </Editable.DraggableAccordionItem>
                                     )}
 
-                                    {showNewLayouts &&
-                                      section.announcements &&
+                                    {section.announcements &&
```

Comment:
is the removal of feature flag here intended?

Reply:
yes!